src/components/dashboard/container/Concepts.jsx

```diff
@@ -7,6 +7,8 @@ import '../styles/index.css';
 
 import SearchConcept from '../components/concepts/Search';
 import ConceptList from '../components/concepts/ConceptList';
+import Paginations from '../../bulkConcepts/component/Paginations';
+
```

Comment:
This whitespace is irrelevant and should be removed.

Reply:
Thanks for that observation let me get rid of the whitespace